src/gui/properties/peerlistwidget.cpp

```diff
@@ -138,6 +136,67 @@ PeerListWidget::~PeerListWidget()
     delete m_copyHotkey;
 }
 
+void PeerListWidget::displayToggleColumnsMenu(const QPoint&)
+{
+    QMenu hideshowColumn(this);
+    hideshowColumn.setTitle(tr("Column visibility"));
+    QList<QAction*> actions;
+    for (int i = 0; i < PeerListDelegate::IP_HIDDEN; ++i) {
+        QAction *myAct = hideshowColumn.addAction(this->headerData(i, Qt::Horizontal, Qt::DisplayRole).toString());
+        myAct->setCheckable(true);
+        myAct->setChecked(!isColumnHidden(i));
+        actions.append(myAct);
+    }
+    int visibleCols = 0;
+    for (unsigned int i = 0; i < PeerListDelegate::COL_COUNT; i++) {
+        if (!isColumnHidden(i))
+            visibleCols++;
```

Comment:
here visibleCols become greater than zero. Following `if` is redundant. Just do `break` here!